po/pt_BR.po

```diff
@@ -0,0 +1,153 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
This should be replaced with the same header that other po-files use.

Reply:
Considering all the changes made to the template recently, it would be easier to just start from scratch and merge these translations.